### ISSUE TITLE
Please update chart to use redis 14.1. 
X subchart

### ISSUE THREAD

**Issue by jsalatiel** [bootc/netbox-chart#49]
Please update this chart to use redis 14.1.X subchart. Bitnami/redis 14.1.x has fixed lots of problems in Sentinel.

**Comment by modzilla99** [comment]
I tried using the newest redis 14.3.3 chart by changing the Chart.yml, while this works, it does make it so that the netbox.yml points to Port 0. When changing it manually afterwards it will begin to work like normal.

Any ideas @bootc?

**Comment by modzilla99** [comment]
To reproduce:

Chart.yml
```yaml
apiVersion: v2
name: netbox
version: 3.0.0
appVersion: v2.10.4
description: IP address management (IPAM) and data center infrastructure management (DCIM) tool
home: https://github.com/bootc/netbox-chart
icon: https://raw.githubusercontent.com/netbox-community/netbox/develop/docs/netbox_logo.png
maintainers:
- name: Chris Boot
  email: bootc@boo.tc
  url: https://github.com/bootc
dependencies:
- name: postgresql
  version: 10.x.x
  repository: https://charts.bitnami.com/bitnami
  condition: postgresql.enabled
- name: redis
  version: 14.3.x
  repository: https://charts.bitnami.com/bitnami
  condition: redis.enabled
```

```bash
helm install --dry-run --set tasksRedis.port=6379 netbox netbox-chart/ | grep -A10 REDIS:
```

**Comment by modzilla99** [comment]
I am lost. It doesn't even work upstream:

```yaml
# helm install --dry-run --repo https://charts.boo.tc --set cachingRedis.port=234 --set Redis.port=6378 netbox netbox | grep -A10 REDIS:
    REDIS:
      tasks:
        HOST: "netbox-redis-master"
        PORT: 6379
        DATABASE: 0
        SSL: false
      caching:
        HOST: "netbox-redis-master"
        PORT: 6379
        DATABASE: 1
        SSL: false
```
or
```yaml
# cat v1.yml
tasksRedis:
  port: 6378

cachingRedis:
  port: 123456
# helm install --dry-run --repo https://charts.boo.tc --values v1.yml netbox netbox | grep -A10 REDIS:
    REDIS:
      tasks:
        HOST: "netbox-redis-master"
        PORT: 6379
        DATABASE: 0
        SSL: false
      caching:
        HOST: "netbox-redis-master"
        PORT: 6379
        DATABASE: 1
        SSL: false
```

**Comment by bootc** [comment]
The ConfigMap refers to:
```yaml
    REDIS:
      tasks:
[...]
        PORT: {{ .Values.redis.redisPort | int }}
```

This matches how you change the port number in version 12.x of the Bitnami chart, so you should be able to change the port to match by using that value even though it's ignored by the Redis chart.

I'll probably upgrade to a newer Redis chart soon, but it'll mean a major version bump for the NetBox chart too so I'm not going to undertake this lightly.

**Comment by bootc** [comment]
I've just released version [4.0.1](https://github.com/bootc/netbox-chart/releases/tag/4.0.1) of this chart which resolves this issue.
